.eslintrc.js

```diff
@@ -5,12 +5,22 @@ module.exports = {
     "prettier"
   ],
   rules: {
-    "no-var": 0,
-    "max-len": 0,
+    "max-len": "off",
+    "strict": "error",
     "prettier/prettier": "error",
   },
   env: {
     node: true,
-    mocha: true
-  }
+  },
+  parserOptions: {
+    sourceType: "script",
+  },
+  overrides: [
+    {
+      files: ["test/**/*"],
+      env: {
+        mocha: true
```

Comment:
Scoping this env just to tests so that we don't accidentally use these globals in the source code.